lib/linter-plus.coffee

```diff
@@ -79,21 +63,37 @@ class Linter
   getLinters: ->
     @linters
 
+  setMessages: (linter, messages) ->
+    @messages.set(linter, messages)
+
+  deleteMessages: (linter, messages) ->
```

Comment:
`messages` is unused. Also, it behaves more like `clear` than `delete`, so I would advise `clearMessages`, maybe.